.github/workflows/publish.yml

```diff
@@ -26,6 +27,5 @@ jobs:
           TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
         run: |
-          poetry version ${{ github.event.release.tag_name }}
-          poetry build
-          poetry run twine upload dist/*
+          python -m poetry build
```

Comment:
Why all the python -m prefixes after installing poetry? It should just be in local bin and on the path, so I if possible I would rather not add the explicit python call.